### ISSUE TITLE
Dev: fix Alert: Orphaned form label on Security page

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/tdm-calculator#3038]
### Overview
We need to fix the Alert: Orphaned form label on Security so that page can be WCAG compliant.

#### Details
Category: Alert
Name: Orphaned form label
(The Accessibility wiki for this Alert has a solution)

Page that will be worked on:
Security
### Action Items
- [x] Review wiki page: Accessibility, How to run the WAVE browser extension (Resource 1.01)
   - [x] Install the WAVE browser extension
- [x] Log in as the user indicated (resource 1.02.01) 
- [x] navigate to the page you are going to fix (resource 1.02.02)
- [x] Open the TDM WAVE Accessibility Audit Spreadsheet, for this Alert (resource 1.03) and find the row that relates to this  issues page.
- [x] Run the WAVE extension with no elements focused, or clicked
- [x] Find the Alert referenced in the Spreadsheet (1.03)
- [x] This alert has been resolved on other pages, so review `Project Team Solution` section (resource 1.04.01.01)
- [x] Apply solution
- [x] Submit PR
- [x] Add PR to spreadsheet in PR column next to Alert (resource 1.03)
- [x] On this issue
   - [x] Add labels: 
      - `ready for accessibility lead`
   - [x] Change status to: `Technical Review`
   - to the epic #2896
     - [x] Add the label `ready for product`
      - [x] Add this comment 
         ```
          - #3038 is closed.  Please review to see if there are any remaining instances of this problem
         ```

### Resources/Instructions
- 1.01[How to run the WAVE browser extension](https://github.com/hackforla/accessibility/blob/main/docs/getting-started/wave-browser-extension.md)
- 1.02 Application details
   - 1.02.01 User Profile Required: Security Admin
   - 1.02.02 Page URL: https://tdm-dev.azurewebsites.net/roles
      - Requires sign in: TRUE
      - Can access page directly from URL: FALSE
- 1.03 [TDM WAVE Accessibility Audit Spreadsheet, tab: 2. Audit Log, filter: Alert: Orphaned form label](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0&fvid=336057419)
   -  Spreadsheet Row: 3
- 1.04 [Accessibility Known Issues Index](https://github.com/hackforla/accessibility/blob/main/docs/known-issues-and-solutions/solutions-index/index.md)
    - 1.04.01 [WAVE Known Accessibility Issue: Alert - Orphaned form label](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/alert-orphaned-form-label)
        - 1.04.01.01 [WAVE Known Accessibility Issue: Alert - Orphaned form label, section: Project Team Solution](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/alert-orphaned-form-label/#project-team-solution)

Additional reading (optional)
- https://hackforla.github.io/accessibility/

#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - ~[Staging Deck, slide 3038](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g39673b86911_4_14#slide=id.g39673b86911_4_14)~
- Release deck
   - [2026-MM-DD Release deck, slide 3038]

**Comment by Rabia2219** [comment]
@geolunalg 
Please add a level label. You can choose from one of the following:

- `good first issue`
- `level: easy`
- `level: medium`
- `level: hard`

Once you have added the level label, please:
- remove the `level: missing` label
- remove the `ready for accessibility lead` label
- add the `ready for prioritization` label

**Comment by Philc90** [comment]
Verified this Alert no longer appears on the Security page, appears to have been fixed already.

<img width="1881" height="901" alt="Image" src="https://github.com/user-attachments/assets/df594408-1ffa-4e90-bfee-dc5c442028b3" />

**Comment by geolunalg** [comment]
This was address in the first round of accessability. This issue can be close without a PR.
`- #3038 is closed.  Please review to see if there are any remaining instances of this problem`

**Comment by ExperimentsInHonesty** [comment]
@geolunalg @Philc90 
- I updated the spreadsheet, and am closing this issue. The fix was made on issue #2709.  Contrary to instructions below do not expand scope of this issue

>- #3038 is closed.  Please review to see if there are any remaining instances of this problem
